scripts/jenkins/ardana/ansible/deploy-crowbar.yml

```diff
@@ -30,6 +30,15 @@
             name: crowbar_setup
           vars:
             qa_crowbarsetup_cmd: onadmin_batch
+
+        - include_role:
```

Comment:
Just curious, why do you need to reboot the cloud nodes after deploying a cloud with maintenance updates ?

Reply:
BTW Well itis a good idea to reboot cloud after u installed it. To be sure that installed maintenance updates will not break booting process or other components. Right now I can think of some "wrong" post-install operations or missing linking/unlinking, sockets, used old libraries, etc.).

Reply:
ok, I understand and I agree with you